### DISCUSSION TITLE
SSH host aliases and repo/propose commands

### DISCUSSION THREAD

**Discussion by c17r** [git-town/git-town#3235] [Q&A] [answered]
I have my ~/.ssh/config file setup similar to
```
Host CorpHub
    HostName github.com
    User git
    IdentityFile "<path to Corp key>"

Host GitHub
    HostName github.com
    User git
    IdentityFile "<path to Personal key>"
```

so I can `git checkout CorpHub:<org>/<repo>` and it works

git-town _mostly_ works with this except for things that open browser windows as it'll try to do `https://corphub/<org>/<repo>...` which obviously doesn't work since it's just an SSH alias and not true DNS alias.

Is there anyway around this currently? My git-town settings are:
```
git-town.hosting-platform=github
git-town.main-branch=main
git-town.perennial-branches=
git-town.ship-delete-tracking-branch=false
git-town.sync-feature-strategy=rebase
```
I had `origin-hostname=CorpHub` and had the issue. I removed it but the autodetect has the same issue.

**Comment by kevgo** [comment, marked as answer]
Have you tried this: https://www.git-town.com/preferences/hosting-origin-hostname

**Reply by c17r** [reply]
I had that originally set to `CorpHub` and I had this issue. I unset it but the autodetect still does the same thing. 

Should I be setting it to `github.com`? Because it says "The given value should match the hostname in your SSH config file"

**Reply by c17r** [reply]
Looking back, I guess I was reading it too quickly. The documentation does explicitly say *hostname* in your SSH which means, yes, it should be `github.com`. I have set it and things works. Thanks for being patient :)
